packages/kit/test/apps/hooks/src/hooks.js

```diff
@@ -0,0 +1,39 @@
+// The following line does not work with vite resolution. Why?
+// import { sequence } from '@sveltejs/kit/hooks';
+import { sequence } from '../../../../dist/hooks';
```

Comment:
better to test source code than build output, where possible

```suggestion
import { sequence } from '../../../../src/runtime/hooks';
```

Reply:
Applied